runtime/src/main/java/io/micronaut/jackson/JacksonPropertyNamingStrategy.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017-2018 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.jackson;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+
+public enum JacksonPropertyNamingStrategy {
```

Comment:
Why does this class need to exist? You can just use the jackson type in the config class